src/Store/AppReducer.js

```diff
@@ -17,6 +18,7 @@ export function systemReducer(cols, INVENTORY_ACTION_TYPES) {
         })),
       };
     },
+    // This is elsewhere too. only use one.
```

Comment:
im not sure if this is outdated now, but I dont see where else this is used? Its 'defined' here and only used in inventory